lib/Basics/StringUtils.cpp

```diff
@@ -25,6 +25,14 @@
 
 #include <stdio.h>
 #include <ctype.h>
+#include <vector>
+#include <bitset>
+#include <cstdint>
+#include <string>
```

Comment:
cstdint and string are included by Common.h already for all files - can be removed here.